src/main/java/com/microsoft/aad/adal4j/AuthenticationAuthority.java

```diff
@@ -47,6 +47,7 @@
     private final static String DISCOVERY_ENDPOINT = "common/discovery/instance";
     private final static String TOKEN_ENDPOINT = "/oauth2/token";
     private final static String USER_REALM_ENDPOINT = "common/userrealm";
+    final static String DEVICE_CODE_ENDPOINT = "/oauth2/devicecode";

```

Comment:
Should this be private like all of the other endpoints? Only being used in unit tests.